data/autoyast_sle15/autoyast_media_install_textmode_allpatterns_aarch64.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
```

Comment:
The name should not so general, this may not work for all SLE15 SP*, suggest to name it as data/autoyast_sle15/autoyast_sles15sp4_media_install_textmode_allpatterns_minimal_aarch64.xml It's a specific profile for minimal update, so add 'minimal' there.  WDYT?

We have another ticket to move all profiles to data/yam without any subfolder, so the name with all info will be better.

Same for the other profile files.

Reply:
Thanks for review, I update the profile name. The qcow in created for migration minimal update cases, I add this sentence to the autoyast installation testsuit description instead of changing the profile name, because the installation didn't do any minimal update thing during installation.